pandas/core/generic.py

```diff
@@ -2634,7 +2634,8 @@ def to_sql(
             `index` is True, then the index names are used.
             A sequence should be given if the DataFrame uses MultiIndex.
         chunksize : int, optional
-            Specify the number of rows in each batch to be written at a time.
+            If batching is enabled via `method`, specify the number of rows in
```

Comment:
Not overly familiar with our implementation but I *think* that this isn't tied to method as described here - are you seeing something definitely in the code that leads you to believe this?

@jorisvandenbossche

Reply:
I commented on the issue about it: https://github.com/pandas-dev/pandas/issues/35891, let's discuss there further (but indeed it doesn't seem to depend on the method based on the code)